src/context/walletContext/walletContext.tsx

```
@@ -0,0 +1,180 @@
+                    setWallets(wallets)
+                    setIsFetchingWallets(false)
+                    return wallets
+                } else {
+                    console.warn('Failed to fetch user wallets.')
+                }
+            }
+        } catch (error) {
+            console.error('ERROR WHEN FETCHING USER', error)
+        } finally {
+            setIsFetchingWallets(false)
+            return null
```

Biome

[error] Unsafe usage of 'return'.

'return' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)